RELEASES.md

```diff
@@ -34,6 +34,42 @@ python "${SOURCE_DIR}/build_tools/fileset_tool.py artifact-flatten *.tar.xz -o o
 popd
 ```
 
+### From `install_rocm_from_artifacts.py`
+
+This script installs ROCm community builds produced by TheRock from either a developer/nightly release, a specific CI runner build or an already existing installation of TheRock. This script is used by CI and can be used locally.
+
+Script setup:
+
+```
+python3 -m venv venv && source venv/bin/activate
```

Comment:
Add `.` prefix to `venv` directory name, matching what we include in `.gitignore`: https://github.com/ROCm/TheRock/blob/5c25161135d0afd79f2dffe1d3ad341663ad385f/.gitignore#L14

```suggestion
python3 -m venv .venv && source .venv/bin/activate
```